app/models/work.rb

```diff
@@ -0,0 +1,29 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, through: :votes, dependent: :destroy
+  validates :title, presence: true
+
+  def self.get_media_categories
+    books = self.get_media("books")
+    albums = self.get_media("albums")
+    movies = self.get_media("movies")
```

Comment:
Here you search for works with category `albums`, `books` and `movies`, but your database seeds and the dropdown menu on your new work form use the singular version of these (`album`, `book`, `movie`). As a result, your index and splash pages don't display any works.